zora/es.js

```diff
@@ -0,0 +1 @@
+export * from './dist/index.cjs';
```

Comment:
I am afraid this does not work for the browser. I tried to import this file from a browser but the .cjs extension makes it think the mime/time is not js

It means I can't write something like

```Javascript
  import {test} from 'https://unpkg.com/zora@5.0.3/dist/es.js';

    test('foo', t => {
        t.ok(true)
    })
```

We still need to provide a "bundled" ES version in the package so CDNs can work out of the box (this is something I want to keep)